### DISCUSSION TITLE
'comparator' issue

### DISCUSSION THREAD

**Discussion by dannykspaw** [apache/superset#12356] [Q&A / Help]
Everything gets setup perfectly fine on the chart side. I'll then save the chart and publish it to dashboard and I get an error on the dashboard. See below:

![Screen Shot 2021-01-08 at 12 02 41 PM](https://user-images.githubusercontent.com/49650445/104048768-6cb42400-51a9-11eb-88b2-9ebbbc2590be.png)

Anyone else have this problem? If so, advice on how to solve?

**Comment by tlhinman** [comment]
We had this problem and removing the IS NOT NULL filter seems to alleviate it. Of course, we're not sure WHY that filter is a problem, nor how we can use it without encountering the error.

**Comment by kamalkeshavani-aiinside** [comment]
Please refer this https://github.com/apache/superset/issues/12755
The issue is resolved in master branch.

**Comment by villebro** [comment]
@dannykspaw this has been cherry picked for the `1.0.1` version of Superset which is scheduled to be released shortly!
